lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,19 @@
+namespace :broadcasts do
+  desc "Send Welcome Notifications once a day"
+  task send_welcome_notification_flow: :environment do
+    # In order to prevent new users from receiving multiple welcome notifications in a day,
+    # a feature_live_date is required. The script will only be effective after feature_live_date
+    # and will ultimately be superseded by 7.days.ago when it's larger than feature_live_date.
```

Comment:
@vaidehijoshi re:
 > same comment here re: date. what date would this actually be?

I also got a little confused while adding this block of comment. Let me know if it can/should better be worded